File: CompSci 1/Unit 7, Math Class/Pythagorean Theorem/Pythagorean.java
import java.util.Scanner;

public class Pythagorean {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = 0;
        int b = 0;
        double c = 0;

        System.out.print("Enter side 1 --> ");
        a = sc.nextInt();
        System.out.print("Enter side 2 --> ");
        b = sc.nextInt();

        c = Math.sqrt(Math.pow(a, 2) + Math.pow(b, 2));

        System.out.println("\nThe hypotenuse of the triangle = " + Math.round(c));
    }
}

File: CompSci 1/Unit 7, Math Class/Pythagorean Theorem/Pythagorean.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    var a = 0
    var b = 0
    var c = 0.0

    print("Enter side 1 --> ")
    a = sc.nextInt()
    print("Enter side 2 --> ")
    b = sc.nextInt()

    c = Math.sqrt(Math.pow(a.toDouble(), 2.0) + Math.pow(b.toDouble(), 2.0))

    println("\nThe hypotenuse of the triangle = " + Math.round(c))
}
